Cargo.lock

```diff
@@ -1,5 +1,7 @@
 # This file is automatically @generated by Cargo.
 # It is not intended for manual editing.
+version = 3
+
```

Comment:
Why was this changed. Did you build with nightly cargo.

Reply:
I did, it fixed itself when I added the bootstrap test and ran it properly.